app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -101,22 +133,26 @@ def parse_az(az)
       end
 
       def parse_instance(instance)
-        uid         = "#{@subscription_id}\\#{instance.fetch_path('resourceGroup')}\\#{instance.fetch_path('name')}"
+        uid = resource_uid(@subscription_id,
+                           instance.fetch_path('resourceGroup'),
+                           instance.fetch_path('type').downcase,
+                           instance.fetch_path('name'))
```

Comment:
For the record - discussed this issue with @bzwei. A separate PR will be made to set the instance ems_ref and uid_ems value to the GUID exposed by the model view in the azure-armrest gem. However, the parse_instance method will continue to return the UID composed by this PR.